sros2/sros2/policy/templates/dds/permissions.xsl

```diff
@@ -57,6 +59,21 @@
                   </xsl:for-each>
                 </allow_rule>
               </xsl:if>
+              <xsl:if test="$allow_ros_discovery_topic">
+                <allow_rule>
+                  <xsl:copy-of select="$template_domains"/>
+                  <publish>
+                    <topics>
+                      <topic>ros_discovery_info</topic>
+                    </topics>
+                  </publish>
+                  <subscribe>
+                    <topics>
+                      <topic>ros_discovery_info</topic>
+                    </topics>
+                  </subscribe>
+                </allow_rule>
+              </xsl:if>
```

Comment:
Lets hope we don't have to keep this fully connected data flow rule forever.

Reply:
Yes. I think that in a near future (one/two months), all the `rmw` implementation will be migrated.

Reply:
I believe what @ruffsl mean is that hopefully this will be done out of band and not require a fully connected rule on this topic.

Reply:
Oh, I see. I think it's definitely possible.
It would be great to finish discussing this alternative in the design document, before the other migrations are done.
Of course, we can first migrate all to use the `ros_discovery_info` topic, and then change them again to use `Participant` `userData`.